libs/components/forms/src/lib/modules/form-error/form-errors.component.ts

```diff
@@ -30,12 +29,6 @@ import { SkyFormErrorComponent } from './form-error.component';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class SkyFormErrorsComponent {
-  /**
-   * The validation errors from the form control.
-   */
-  @Input()
-  public errors: ValidationErrors | null | undefined;
```

Comment:
`sky-form-errors` will no longer be handling any errors so this input is no longer needed